requirements.txt

```diff
@@ -2,8 +2,9 @@
 -e .
 
 # Additional requirements for running the testsuite and development
-flake8
-pytest
-pytest-cov
+pycodestyle==2.2.0
+flake8==3.2.0
+pytest==3.0.4
+pytest-cov==2.4.0
```

Comment:
Actually, I am wondering about this. This could create conflicts when installing geoalchemy2 along other packages in a Python environment, no?